src/test/java/uk/gov/justice/probation/courtcasematcher/service/MatcherServiceTest.java

```diff
@@ -259,6 +308,7 @@ void givenZeroMatches_whenSearchResponse_thenReturn() {
         assertThat(updatedCase.getDefendants().get(0).getGroupedOffenderMatches().getMatches()).isEmpty();
         assertThat(updatedCase.getDefendants().get(1).getGroupedOffenderMatches().getMatches()).isEmpty();
 
+        verifyNoInteractions(personMatchScoreRestClient);
         verify(telemetryService).trackOffenderMatchEvent(FIRST_DEFENDANT, COURT_CASE, noMatches);
         verify(telemetryService).trackOffenderMatchEvent(SECOND_DEFENDANT, COURT_CASE, noMatches);
     }
```

Comment:
Can we add a test for when the call to person match score fails? We want to be sure that the rest of the process continues without any issues if that's the case.

Reply:
added ta